### PR TITLE
feat: tabs example in build journey

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Adds tabs in build journey > main settings 
- Transforms ::: into includes

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 1]
```diff
@@ -6,165 +6,166 @@ namespace: docs_guides_build_configure_main_settings
...
 
-In this section, you'll modify some configurations of your application. You can see if the changes ocurred by [accessing your Azion domain](/en/documentation/products/guides/configure-a-domain/).
+<Tabs client:visible>
+	<Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+	<Fragment slot="tab.api">API</Fragment>
 
-1. [Access Azion Console](https://console.azion.com).
-2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and select **Edge Application**.
-3. Click the edge application you want to configure. From this screen, you can alter the application's name, delivery settings, and which modules it uses.
+	<Fragment slot="panel.rtm">
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:26]: Aqui deveria ser: <Fragment slot="panel.console">Console</Fragment>?

Comment (bru-andrade) [reply src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:26]: verdade... vou mudar para o link de RTM, qnd vcs tiverem um padrão pro console, aí é só adicionar na nova aba

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 2]
```diff
@@ -6,165 +6,166 @@ namespace: docs_guides_build_configure_main_settings
...
+        1. [Access Azion Console](https://console.azion.com).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:27]:
```suggestion
        1. [Access Real-Time Manager](https://manager.azion.com).
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx [hunk 3]
```diff
@@ -6,169 +6,171 @@ namespace: docs_guides_build_configure_main_settings
...
-
-Os próximos passos irão guiá-lo na edição das configurações de uma edge application boilerplate. Sinta-se à vontade para alterar essas configurações para atender às suas necessidades.
-
-### Delivery protocols
+<Tabs client:visible>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+	<Fragment slot="tab.api">API</Fragment>
 
-Modifique sua aplicação para suportar os [protocolos de entrega](/pt-br/documentacao/produtos/build/edge-application/main-settings/#protocolos-de-entrega) disponíveis.
+    <Fragment slot="panel.rtm">
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx:22]: same

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx [hunk 4]
```diff
@@ -6,169 +6,171 @@ namespace: docs_guides_build_configure_main_settings
...
-6. Clique no botão **Save**.
+    Os [Módulos de Edge Application](/pt-br/documentacao/produtos/edge-application/#modules) podem desbloquear funcionalidades e recursos adicionais para suas aplicações. Veja a [página de preços](https://www.azion.com/pt-br/pricing/) para mais detalhes.
 
-:::note[Opcional]
-Você também pode aproveitar esta oportunidade para atualizar seu método de pagamento na aba **Payment Methods**.
-:::
+    Por enquanto, você ativará dois módulos. Os módulos **Application Accelerator** e **Edge Functions** serão usados quando você [ajustar suas configurações de cache](/pt-br/documentacao/produtos/guias/build/ajustar-cache-settings/), [instanciar uma edge function](/pt-br/documentacao/produtos/guias/build/instanciar-edge-functions/) e [Trabalhar com Rules Engine](/pt-br/documentacao/produtos/guias/build/trabalhar-com-rules-engine/). 
 
-Em seguida, para habilitar os módulos para o sua aplicação:
+    Se você é um [accont owner](/pt-br/documentacao/produtos/guias/users-management/), comece ativando os seguintes módulos para sua conta:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx:73]:
```suggestion
    Se você é um [account owner](/pt-br/documentacao/produtos/guias/users-management/), comece ativando os seguintes módulos para sua conta:
```

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 5]
```diff
@@ -6,165 +6,166 @@ namespace: docs_guides_build_configure_main_settings
...
+
+        ### Modules
+
+        [Modules](/en/documentation/products/build/edge-application/#modules) can unlock additional functionalities and features for your applications. See the [Pricing page](https://www.azion.com/en/pricing/) for more details.
+
+        For now, you'll enable two modules. The **Application Accelerator** and **Edge Functions** modules will be used for when you [tune your cache settings](/en/documentation/products/guides/build/tune-cache-settings/), [instantiate an edge function](/en/documentation/products/guides/build/instantiate-edge-functions/), and [work with Rules Engine](/en/documentation/products/guides/build/work-with-rules-engine/). 
+
+        If you're an [Account owner](/en/documentation/products/guides/users-management/), start by enabling the following modules for your account:
+
+        1. [Access Console](https://console.azion.com).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:77]:
```suggestion
        1. [Access Real-Time Manager](https://manager.azion.com).
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx [hunk 6]
```diff
@@ -6,169 +6,171 @@ namespace: docs_guides_build_configure_main_settings
...
-Os próximos passos irão guiá-lo na edição das configurações de uma edge application boilerplate. Sinta-se à vontade para alterar essas configurações para atender às suas necessidades.
-
-### Delivery protocols
+<Tabs client:visible>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+	<Fragment slot="tab.api">API</Fragment>
 
-Modifique sua aplicação para suportar os [protocolos de entrega](/pt-br/documentacao/produtos/build/edge-application/main-settings/#protocolos-de-entrega) disponíveis.
+    <Fragment slot="panel.rtm">
+    1. [Acesse o Azion Console](https://console.azion.com).
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx:23]:
```suggestion
    1. [Acesse o Real-Time Manager](https://manager.azion.com).
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx [hunk 7]
```diff
@@ -6,169 +6,171 @@ namespace: docs_guides_build_configure_main_settings
...
+    Por enquanto, você ativará dois módulos. Os módulos **Application Accelerator** e **Edge Functions** serão usados quando você [ajustar suas configurações de cache](/pt-br/documentacao/produtos/guias/build/ajustar-cache-settings/), [instanciar uma edge function](/pt-br/documentacao/produtos/guias/build/instanciar-edge-functions/) e [Trabalhar com Rules Engine](/pt-br/documentacao/produtos/guias/build/trabalhar-com-rules-engine/). 
 
-Em seguida, para habilitar os módulos para o sua aplicação:
+    Se você é um [accont owner](/pt-br/documentacao/produtos/guias/users-management/), comece ativando os seguintes módulos para sua conta:
 
-7. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Edge Application**.
-8. Clique na edge application que você deseja configurar.
-9. Ative os módulos **Application Accelerator** e **Edge Functions**.
-10. Clique no botão **Save**.
+    1. [Acesse o Azion Console](https://console.azion.com).
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/main-settings.mdx:75]:
```suggestion
    1. [Acesse o Real-Time Manager](https://manager.azion.com).
```

**Pull request merged**